lib/iris/tests/unit/test_Future.py

```diff
@@ -43,13 +43,17 @@ def test_invalid_attribute(self):
         with self.assertRaises(AttributeError):
             future.numberwang = 7
 
-    def test_invalid_netcdf_promote_attributes(self):
+    def test_netcdf_promote(self):
         future = Future()
-        states = [True, False]
-        exp_emsg = "deprecated 'netcdf_promote' behaviour has been removed"
-        for state in states:
-            with self.assertRaisesRegexp(AttributeError, exp_emsg):
-                future.netcdf_promote = state
+        exp_emsg = "'Future' property 'netcdf_promote' is deprecated"
+        with self.assertWarnsRegexp(exp_emsg):
+            future.netcdf_promote = True
+
+    def test_invalid_netcdf_promote(self):
+        future = Future()
+        exp_emsg = "'Future' property 'netcdf_promote' has been deprecated"
+        with self.assertRaisesRegexp(AttributeError, exp_emsg):
+            future.netcdf_promote = False
```

Comment:
These tests are now wrong. The `netcdf_promote` FUTURE has been removed, so if a user tries to set either `True` and `False` for this flag an error should be raised. Please reset these tests to their original behaviour.